e2e_tests/integration/connect-form.spec.ts

```diff
@@ -105,5 +111,68 @@ describe('Connect form', () => {
       cy.connect('neo4j', Cypress.config('password'), boltUrl)
       cy.executeCommand(':server disconnect')
     })
+
+    if (isEnterpriseEdition()) {
+      it.only('shows correct metadata when using db field', () => {
```

Comment:
No `.only` please

Reply:
I'll set up an on commit hook for this so I won't forget it again